packages/react-components/react-color-picker-preview/library/docs/Spec.md

```diff
@@ -2,62 +2,458 @@
+| ColorSlider allows user to pick a color using individual channel         | ColorSlider    | ColorSlider | ⚠️        |
+| AlphaSlider allows user to pick a color using alpha channel              |                | AlphaSlider | ❌        |
 
 ## Sample Code
 
-_Provide some representative example code that uses the proposed API for the component_
+```jsx
+import {
+  ColorPicker,
+  ColorArea,
+  ColorSliderProps,
+  AlphaSlider,
+  HueSlider,
+  ColorPickerOnSelectEventHandler,
+} from '@fluentui/react-color-picker-preview';
+
+export const Default = () => {
+  const COLOR = 'rgba(0, 255, 170, 1)';
+  const [selectedColor, setSelectedColor] = React.useState(COLOR);
+  const handleChange: ColorPickerOnSelectEventHandler = (_, data) => {
+    setSelectedColor(data.selectedColorHex);
+  };
+
+  return (
+    <>
+      <ColorPicker color={selectedColor} onChange={handleChange}>
+        <ColorArea />
+        <AlphaSlider />
+        <HueSlider />
+      </ColorPicker>
+      <div style={{ backgroundColor: `${selectedColor}` }} />
+    </>
+  );
+};
+```
 
 ## Variants
 
-_Describe visual or functional variants of this control, if applicable. For example, a slider could have a 2D variant._
+### Slider variants
+
+- horizontal (default)
+- vertical
+
+ColorSliders might represent different color channels.
+
+### Shapes
+
+- `square` (default)
+- `rounded`
+
+### Size
+
+Size will be the same as in Office products.
+For custom sizes users might want to customize it via CSS.
+
+### States
+
+- `rest`
+- `focused`
+
+### Color support
+
+- RGB(A)
+- Hex
+- HSL(A)
+- HSV/HSB
 
 ## API
 
-_List the **Props** and **Slots** proposed for the component. Ideally this would just be a link to the component's `.types.ts` file_
+### ColorPicker
+
+| Property   | Values              | Default   | Purpose                                |
+| ---------- | ------------------- | --------- | -------------------------------------- |
+| color      | `string`            |           | Sets color value                       |
+| onChange   | `function`          | undefined | Callback called when color is selected |
+| shape      | `square`, `rounded` | `square`  | Sets shape                             |
+| step       | `number`            | 1         | Step for arrow navigation              |
+| customStep | `number`            | 3         | Fast navigation                        |
+
+Input fields with color values will be in `renderUtils`. It will contain default ColorPicker which has all colors represented and a preview swatch. This functionality is similar to V8.
+
+```ts
+import { ColorPickerLayout } from '@fluentui/react-color-picker-preview';
+```
+
+```tsx
+export const Default = () => {
+  const COLOR = 'rgba(0, 255, 170, 1)';
+  const [selectedColor, setSelectedColor] = React.useState(COLOR);
+  const handleChange: ColorPickerOnSelectEventHandler = (_, data) => {
+    setSelectedColor(data.selectedColorHex);
+  };
+
+  return (
+    <>
+      <ColorPickerLayout
```

Comment:
How does this relate to the ColorPicker in line 89? Are these both suggested APIs?

Reply:
I'd say it's a helper wrapper. Ideally if partners use JSX composition. But for those who wants similar functionality to V8 we can provide this helper layout